.azure-pipelines/ultimate-pipeline.yml

```diff
@@ -804,15 +805,25 @@ stages:
           parameters:
             artifact: build-linux-arm64-working-directory
 
-        - template: steps/run-in-docker.yml
-          parameters:
-            build: true
-            baseImage: $(baseImage)
-            command: "BuildLinuxIntegrationTests --framework $(publishTargetFramework)"
-
+        # we build the integration tests in parallel to starting the dependencies
+        # the first command is just a copy past of the run-in-docker template
```

Comment:
Urgh, not a fan of this copy-paste I had to make big changes to this in the ci-app branch, and this risks missing stuff 🙁 Also the docker-compose build step relies on the output of this stage, so I don't think we can parallelize it anyway (though the `docker-compose build` step _should_ essentially be a noop 🤔 

Parallelizing the `docker-compose build` and `StartDependencies` looks fine to me though 👍

Reply:
Anyway it didn't bring much for ARM. We could rediscuss once we compile the samples beforehand, if we want to do that, in which case we coud try to do it more cleanly.

Reply:
Fixed in 3c899690453315bbb2a45dbbd6e33fa05061aa89